gluten-flink/docs/Flink.md

```diff
@@ -91,24 +91,51 @@ Submit test script from `flink run`. You can use the `StreamSQLExample` as an ex
 ### Flink local cluster
 
 After deploying flink binaries, please add gluten-flink jar to flink library path,
-including gluten-flink-runtime-1.4.0.jar, gluten-flink-loader-1.4.0.jar and Velox4j jars above.
+including gluten-flink-runtime-1.5.0.jar, gluten-flink-loader-1.5.0.jar and Velox4j jars above.
+Take following command as an example:
```

Comment:
Let's make it concise. Suggestion for the above:

After deploying Flink binaries, please configure the paths of gluten-flink jars and the dependency jars for Flink to use, as follows: